packages/relay-runtime/network/RelayNetworkLogger.js

```diff
@@ -55,9 +55,13 @@ const RelayNetworkLogger = {
       console.time && console.time(idName);
 
       const onSettled = (error: ?Error, response: ?QueryPayload): void => {
-        console.groupCollapsed(`%c${idName}`, error ? 'color:red' : '');
+        if (console.groupCollapsed) {
+          console.groupCollapsed(`%c${idName}`, error ? 'color:red' : '');
```

Comment:
I imported this to quickly. If `groupCollapsed` is missing, this might be a bit too verbose. Do you think we should just guard at the top and return the non-wrapped function so it works in environments like Jest or is the ignoring of grouping more helpful?